api/src/pcapi/core/offers/api.py

```diff
@@ -24,13 +24,20 @@
 import pcapi.core.bookings.repository as bookings_repository
 from pcapi.core.categories import subcategories
 import pcapi.core.criteria.models as criteria_models
+from pcapi.core.educational import exceptions as educational_exceptions
 from pcapi.core.educational import models as educational_models
-from pcapi.core.educational.api import offer as educational_api_offer
+from pcapi.core.educational import validation as educational_validation
+from pcapi.core.educational.api import offer as offers_api
```

Comment:
(nommage) `offers_api` est le raccourci habituel pour `core.offers.api`

```suggestion
from pcapi.core.educational.api import offer as educational_offer_api
```